integration_test/src/token_bridge/transfer_token_test.ts

```diff
@@ -1,12 +1,50 @@
 import { execSync } from 'child_process'
+import { AlephiumBridgeChain } from '../alph'
 import { BridgeChain } from '../bridge_chain'
-import { getBridgeChains, randomBigInt } from '../utils'
+import { assert, getBridgeChains, randomBigInt } from '../utils'
 import { TransferTokenTest } from './transfer_token'
+import { default as alephiumDevnetConfig } from '../../../configs/alephium/devnet.json'
+import { ALPHTokenInfo } from 'alephium-wormhole-sdk'
 
-async function attestTokens(alph: BridgeChain, eth: BridgeChain) {
-  const signedVaa0 = await alph.attestToken(alph.testTokenId)
+async function attestInvalidToken(alph: AlephiumBridgeChain) {
+  const testTokenInfo = await alph.getLocalTokenInfo(alephiumDevnetConfig.contracts.testToken)
+  const testTokenInfos = [
+    {...testTokenInfo, decimals: testTokenInfo.decimals + 1},
+    {...testTokenInfo, symbol: testTokenInfo.symbol + 'XX'},
+    {...testTokenInfo, name: testTokenInfo.name + 'XX'},
+  ]
+
+  const test = async (info: any) => {
+    console.log(`trying to attest invalid token: ${JSON.stringify(info)}`)
+    try {
+      await alph.attestWithTokenInfo(alph.testTokenId, info.decimals, info.symbol, info.name)
+    } catch (error) {
+      console.log(`failed to attest invalid token, error: ${error}`)
+      return
+    }
+    assert(false)
```

Comment:
[Nit] Maybe use [toHaveReturnd](https://jestjs.io/docs/expect#tohavereturned)  for `alph.attestWithTokenInfo`?

Reply:
This test is used to test that when we try to attest an invalid token, we cannot get the vaa, so here the function `attestWithTokenInfo` will not return but will throw an error.

In addition, the current integration test does not depend on the test framework 😅